modules/release-notes/pages/relnotes.adoc

```diff
@@ -5,6 +5,42 @@
 
 Take a look at xref:introduction:whats-new.adoc[What's New?] for a list of new features and improvements in this release.
 
+[#release-664]
+== Release 6.6.4 (December 2021)
+
+Couchbase Server 6.6.4 was released in December 2021. 
+This maintenace release includes an important security fix in analytics.
+
+_Quick Links_: <<fixed-issues-664>> | <<common-vulnerabilities-exposures-664>>
+
+[#fixed-issues-664]
+=== Fixed issues
+
+This section highlights the notable issues fixed in this release.
+
+[IMPORTANT]
+====
+This release fixes a high severity vulnerability related to Log4j 2. 
+All users should upgrade to this release as soon as possible.
```

Comment:
Can we change this to
All users utilising the Couchbase Analytics service should upgrade to this release as soon as possible.